receiver/redisreceiver/api_parser.go

```diff
@@ -0,0 +1,39 @@
+package redisreceiver
+
+import "strings"
+
+type apiParser struct {
```

Comment:
Please add comments to explain the purpose of non-trivial structs and functions.

Reply:
Done.